docs/2_2_common_schema.adoc

```diff
@@ -1572,19 +1573,19 @@ _Discussion of the combinations that are allowed_:
 
 >|_[green]#(1)#_
 |_<<fixed>> <<parameter>>_
-|_Non-<<tunable>> <<independent>> <<parameter>>_
+|_Non-<<tunable>> <<parameter>>_
 
 >|_[green]#(2)#_
 |_<<tunable>> <<parameter>>_
-|_<<tunable,`Tunable`>> <<independent>> <<parameter>> (changing such a <<parameter>> triggers an external event (ME) or takes effect at the next communication point (xCS), and <<tunable>> <<calculatedParameter>>/<<output>>/<<local>> variables might change their values)._
+|_<<tunable,`Tunable`>> <<parameter>> (changing such a <<parameter>> triggers an external event (ME) or takes effect at the next communication point (xCS), and <<tunable>> <<calculatedParameter>>/<<output>>/<<local>> variables might change their values)._
```

Comment:
Do you think we should keep the links to the <<parameter>> here? I think these are not meant to link to parameter, because they refer to the tunable parameters.

For instance, the fact that `changing such a <<parameter>> triggers an external event` links to parameter may make the reader thing that there's a parameter associated with each tunable parameter.

I understand if that's not the goal of this PR.

Reply:
A tunable parameter must be independent.

Reply:
Sorry, forgot that the markdown will mess ``<<text like this one>>``.
Updated the original remark.
Anyhow, I understand that this is not the goal of this PR.